concent_api/core/tests/test_integration_force_payment.py

```diff
@@ -393,3 +394,185 @@ def test_provider_send_correct_force_payment_concent_should_accept(self):
                 'timestamp':      self._parse_iso_date_to_timestamp("2018-02-05 12:00:21"),
             }
         )
+
+    def test_provider_send_force_payment_with_wrong_subtask_results_accepted_list_concent_should_return_http_400(self):
```

Comment:
Please make test name more descriptive and explain what is exactly wrong with this list.

Reply:
:+1: